bin/add_frontend_server.rb

```diff
@@ -347,6 +347,11 @@ def generate_instance(environment, instance_provisioning_info, role, instance_ty
     @options['role'] = role
   end
 
+  opts.on('-i', '--instance-type TYPE',
```

Comment:
I'd suggest having a map of instance classifications to instance types - I'm not going to remember that our prod instances are c3.8xlarge or that some smaller instance is what adhocs default to